checkpointer/src/main.rs

```diff
@@ -68,6 +73,24 @@ pub async fn delete_batcher(
     Ok(HttpResponse::Ok().finish())
 }
 
+#[post("/calculate")]
+pub async fn calculate(config: web::Data<Config>) -> Result<impl Responder, Error> {
+    if config
+        .calculate_active
+        .load(std::sync::atomic::Ordering::Relaxed)
```

Comment:
seems fine but SeqCst might be better if it should really never happen twice (but it's a web api so getting their simultaneously will be _really_ tough)